app/views/rent_mailer/rent_created.html.erb

```diff
@@ -0,0 +1,17 @@
+<h1 style='color: green'>A new rent was created!</h1>
+
+<p>
+  <h2>Dear customer <b style='color: blue'><%= @user_renting.first_name+' '+@user_renting.last_name %>,</b></h2>
+  <br>
+  Your rent of the book <b><%= @book_rented.title %></b> has been created at <b><%= Time.zone.now %></b>
```

Comment:
Instead of time.zone.now, use @rent.created_at, since this maybe will be processed in the future, because it's an async job